### DISCUSSION TITLE
Adding evaluation metrics such as F1 score, ROC AUC.

### DISCUSSION THREAD

**Discussion by akashtadwai** [ContinualAI/avalanche#435] [Feature Request]
As continual learning is expanding in its domain from Vision to others. I think including some common metrics such as F1 score, ROC- AUC would be quite useful while experimenting on Imbalanced Datasets.

**Comment by AndreaCossu** [comment]
Your are totally right. We are already working on expanding our base set of metrics (e.g. #212  #213 ). We will definitely take into consideration F1, ROC and AUC.  
If you are willing to help, feel free to get in touch and contribute :smile:
